### PR TITLE
return error for enabling an unknown addon

### PR DESCRIPTION
**Pull request by zhujian7**
Fix: https://github.com/open-cluster-management-io/clusteradm/issues/342

### CODE REVIEW

File: pkg/cmd/addon/enable/exec.go [hunk 1]
```diff
@@ -106,7 +106,22 @@ func (o *Options) runWithClient(clusterClient clusterclientset.Interface,
...
 	}
 
+	cmas, err := addonClient.AddonV1alpha1().ClusterManagementAddOns().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
 	for _, addon := range addons {
+		cmaExist := false
+		for _, cma := range cmas.Items {
```
Comment (qiujian16) [inline pkg/cmd/addon/enable/exec.go:116]: can we just get cma here instead of list? It will require less permission.

Comment (zhujian7) [reply pkg/cmd/addon/enable/exec.go]: PTAL

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**